Core/GraphQL/GraphQLFiles/sell_fragment_order.graphql

```diff
@@ -0,0 +1,94 @@
+fragment FragmentOrder on Order {
+  id
+  orderNumber
+  qrCodeUrl
+  collectionDate
+  status
+  state {
+    workingState
+    status
+    translations {
+      language
+      title
+      desciption
+    }
+  }
+  paymentStatus
+  grossAmount
+  discountAmount
+  netAmount
+  estimatedAt
+  createdAt
+  updatedAt
+  items {
+    id
+    product{...FragmentProduct}
+    productVariant {
+      id
+      externalId
+      price
+      externalId
+      translations {
+        title
+        language
+      }
+    }
+    fulfilmentPoint{ ...FragmentFulfilmentPoint }
+    productModifierItems {
+      quantity
+      productModifierItem {
```

Comment:
We can create `ProductModifierItem` fragment to reuse it here and product.modifierLists.items